index.html

```diff
@@ -115,19 +134,29 @@ <h3>Multi-Post Stories</h3>
                 <li class="padding-right">Ruby</li>
             </ul>
             <button type="button" class="project-button">See Project</button>
+
+            </div>            
+        
         </section> 
     </section>
     <section class="form-container">
-        <h3>CONTACT US</h3>
+        <h3 class="mobH">CONTACT US</h3>
+        <h3 class="deskH">Let`s chat!</h3>
         <P>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </P>
         <form action="https://formspree.io/f/xdoylybq" method="POST">
             <label for="name"></label>
             <input type="text" id="name" name="Full name" placeholder="Full Name" required>
-            <label for="mail"></label>
+            <div class="form-box">
+                <label for="fname"></label>
+                <input type="text" id="fname" name="first name" placeholder="First name" required>
+                <label for="fname"></label>
+                <input type="text" id="lname" name="last name" placeholder="Last name" required>
+                <label for="mail"></label>
+            </div>
```

Comment:
- [ ] There should be one input for Full name (just like the mobile version) and this should take the space of both First Name and Last Name inputs.

Reply:
The desktop version has two Inputs for first name and last name. Are saying i should ignore the desktop inputs and use the mobile version.

Reply:
Hi @Obote, yes the requirements state that you should use one input.

Reply:
Hello, @zeenatLawal am using templet 4 from the list of templet, and according to the requirements, my design should look like the one on the template. So if I remove the two inputs for the first name and last name replacing it with one input tag that captures both names at once this won`t be an issue then.

Reply:
Hi @Obote, Kindly follow this requirement for the contact form
![FireShot Capture 110 - html_css_portfolio_m4_template4_spec md · master · Microverse _ guide_ - gitlab com](https://user-images.githubusercontent.com/46851023/134672725-2b6a3a4e-09f8-44b3-858f-9c7bc6ebb9ce.png)

Reply:
Can i please have a call with me.

Reply:
Hi @Obote, no problem send a link

Reply:
I think that requirement is for the first template and am using the fourth one.

Reply:
Hello @Obote, you can go ahead with the other changes and leave the input as shown in your template.